src/pkg/cli/common.go

```diff
@@ -37,7 +38,6 @@ func PrintObject(root string, data proto.Message) error {
 	if err != nil {
 		return err
 	}
-	// TODO: add color
-	term.Println(string(bytes))
+	term.Println(fmt.Sprintf("%s%s", term.ResetColorStr, string(bytes)))
```

Comment:
@lionello  Is there a nicer way to do this?

Reply:
you shouldn't have to reset color. If you do, it means somewhere else we don't terminate it correctly.